### PR TITLE
feat: DIA-1362: Custom LLM URL support

### PR DESCRIPTION
**Pull request by niklub**
- Add migrations to support Custom LLM provider
- Move key validation to LSE

### CODE REVIEW

**Comment by niklub** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10881478947)
> Successfully merged:  create mode 100644 web/libs/editor/src/tags/control/TimelineLabels.js

**Comment by niklub** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10882383112)
> Successfully merged:  2 files changed, 4 insertions(+), 4 deletions(-)

File: label_studio/ml_model_providers/models.py [hunk 1]
```diff
@@ -59,29 +61,3 @@ def has_permission(self, user):
         return (
             user.is_administrator or user.is_owner or user.is_manager
         ) and user.active_organization_id == self.organization_id
-
-    def validate_api_key(self):
```
Comment (hakan458) [inline label_studio/ml_model_providers/models.py:63]: Why was this funtionality moved to lse_ml_models/serializers.py in separate functions? e.g. `def _validate_openai_connection`

Comment (matt-bernstein) [reply label_studio/ml_model_providers/models.py:63]: makes sense to keep it with all the other logic that's specific to model providers, could go either way

**Review by matt-bernstein — APPROVED** [review]

**Pull request merged**